### DISCUSSION TITLE
Add Cameras Without Detection

### DISCUSSION THREAD

**Discussion by gcortes** [blakeblackshear/frigate#3064] [Ask A Question] [answered]
I have frigate working in docker on a Mac Mini running Ubuntu with three other docker instances running Home Assistant, Node-red, and Mosquitto. I don't have a Coral accelerator nor does it look like I can get one any time soon. I only added one camera and it works fine. I have seven other cameras that I would like to add. My Mac Mini surely would keel over and die doing detection with that many. Is it possible to add the other cameras and set them to not do detection? The reason I want to do so is to just have one source for live views. I have been using Security Spy on another Mac Mini along with its iOS app. I also I tried using the WebRTC app, but has issues connecting, which is strange because the Frigate Card with the same camera works fine

**Comment by NickM-27** [comment, marked as answer]
Hi, yes detection, snapshots, and recording are all completely toggleable and can be disabled by default. They can also be controlled using the HomeAssistant integration in case you wanted to run detection on two or three at a time, but different cameras at different times of the day / night.
